src/kotlin/org/intellij/plugins/hcl/terraform/config/model/TypeModelLoader.kt

```diff
@@ -347,7 +348,24 @@ class TypeModelLoader(val external: Map<String, TypeModelProvider.Additional>) {
     val conflicts: List<String> = value.array<String>("ConflictsWith")?.map { it } ?: emptyList()
 
     val deprecated = value.string("Deprecated")
-    val has_default: Boolean = value.obj("Default")?.isNotEmpty() ?: false
+    val defaultObj = value.obj("Default")
+    val defaultValue = if (defaultObj == null || defaultObj.isEmpty()) {
```

Comment:
Seems `if` could be simplified to just `defaultObj?.let { ... }` since there `null -> null` block in `when` which will sort out `isEmpty` case